pkg/v1/tkg/client/upgrade_cluster.go

```diff
@@ -435,6 +435,8 @@ func (c *TkgClient) applyPatchAndWait(regionalClusterClient, currentClusterClien
 	}
 
 	log.Info("Upgrading control plane nodes...")
+	log.Infof("Attempting to increase kube-vip timeouts")
```

Comment:
I think we should remove this log as it will create confusion to the users running upgrades on `aws`, `azure` and `vsphere without kube-vip` clusters.

We are already adding a log within the other function, so this log should be removed.